planet/planet.ini

```diff
@@ -24,6 +24,7 @@ encoding = UTF-8
 log_level = CRITICAL
 new_feed_items = 2
 activity_threshold = 90
+filter_directories = /usr/share/planet-venus/filters
```

Comment:
This should be `/usr/share/planet-venus/filter` to be able to access common filter files

Reply:
+1